modular/Neu_Food/code/raw/NeuFood_processed.dm

```diff
@@ -13,21 +13,23 @@
 	icon_state = "fat"
 	list_reagents = list(/datum/reagent/consumable/nutriment = SNACK_DECENT)
 	eat_effect = /datum/status_effect/debuff/uncookedfood
-/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/user, params)
+
+/obj/item/reagent_containers/food/snacks/fat/attackby(obj/item/I, mob/living/user, params)
 	var/found_table = locate(/obj/structure/table) in (loc)
 	if(user.mind)
 		short_cooktime = (60 - ((user.mind.get_skill_level(/datum/skill/craft/cooking))*5))
```

Comment:
I feel like all of these should be more like...
```suggestion
		short_cooktime = (6 SECONDS) - (user.mind.get_skill_level(/datum/skill/craft/cooking) * (0.5 SECONDS))
```
which makes it clearer how it works—it's 6 seconds to begin with, and each cooking level reduces it by half a second.
Also I would probably take care to make sure it doesn't drop below 0? I don't want to know what happens if you give a do_after a negative delay.